clients.asciidoc

```diff
@@ -113,10 +113,10 @@ Syncing the Ethereum blockchain is very input-output (I/O) intensive. It is best
 
 *Minimum Requirements:*
 
-* CPU with 2+ cores
-* Solid State Drive (SSD) with at least 50GB free space
-* 4GB RAM minimum, 8GB+ if you have an HDD and not SSD
-* 8 MBit/sec download Internet service
+* CPU with 2+ cores.
+* Solid State Drive (SSD) with at least 50GB free space.
+* 4GB RAM minimum, 8GB+ if you have an HDD and not SSD.
+* 8 MBit/sec download Internet service.
 
 These are the minimum requirements to sync a full (but pruned) copy of an Ethereum-based blockchain.
```

Comment:
We must be very carreful here, a fastsync node does node valid the states after every block like a bitcoin pruned node would do, in only start something like 1000 block before the current state.
Also I'm not sure about the requirement: last week I tryed to sync geth in fast mode, I couldn't even catch up using my dev laptop (i7, 16gb ram) because the i/o are just too much for a hdd

Reply:
just adding dots in this commit
please my suggestion would be to open an issue if you found something to review

Reply:
@SylTi Please add an issue about this. I agree, we need to clarify and address it.